Tests/FunctionalTests/PluginTests.swift

```diff
@@ -188,19 +188,22 @@ class PluginTests: XCTestCase {
     
     func testCommandPluginInvocation() throws {
         try testWithTemporaryDirectory { tmpPath in
-            // Create a sample package with a library target and a plugin.
+            // Create a sample package with a library target and a plugin. It depends on a sample package.
             let packageDir = tmpPath.appending(components: "MyPackage")
-            try localFileSystem.writeFileContents(packageDir.appending(component: "Package.swift")) {
+            try localFileSystem.writeFileContents(packageDir.appending(components: "Package.swift")) {
```

Comment:
?

Reply:
To make it consistent with all the other calls in this function.  This one just happens to have a single component.  I can change it back.

Reply:
Done